### ISSUE TITLE
about gtFine_InstanceIds

### ISSUE THREAD

**Issue by Apsu95** [mcordts/cityscapesScripts#94]
Is it right to treat it as an uint16 image, where first 8 bits represent the label and last 8 bits the instances number? 

please let me know

**Comment by mcordts** [comment]
Nope, please refer to [json2instanceImg.py](https://github.com/mcordts/cityscapesScripts/blob/master/cityscapesscripts/preparation/json2instanceImg.py)
